gw_spaceheat/actors/actor_base.py

```diff
@@ -1,23 +1,21 @@
 from abc import ABC, abstractmethod
 import paho.mqtt.client as mqtt
-import configparser
 import threading
 from typing import List
 from data_classes.sh_node import ShNode
 from actors.mqtt_utils import Subscription, QOS
+import settings
 
 class ActorBase(ABC):
 
     def __init__(self, node: ShNode):
         self.node = node
-        config = configparser.ConfigParser()
-        config.read("config.ini")
-        self.mqttBroker = config["default"]["broker_address"]
+        self.mqttBroker = settings.MQTT_BROKER_ADDRESS
         self.publish_client = mqtt.Client(f"{node.alias}-pub")
-        self.publish_client.username_pw_set(config["default"]["username"], config["default"]["password"])
+        self.publish_client.username_pw_set(settings.MQTT_USER_NAME, None)
```

Comment:
If I'm understanding this correctly, this means that if someone doesn't have `MQTT_USER_NAME` in `settings.py` then it'll default to passing in `None`- is it possible that this will lead to a confusing error down the line (ie something where it isn't immediately clear that the user needs to add `MQTT_USER_NAME` to their settings?)?